vms/vms/settings.py

```diff
@@ -118,6 +118,11 @@
 
 STATIC_ROOT = './static/'
 
+#Instead of sending out real email, during development the emails will be sent
+# to stdout, where from they can be inspected.
+EMAIL_HOST = 'localhost'
+EMAIL_PORT = 1025
+EMAIL_BACKEND = 'django.core.mail.backends.console.EmailBackend'
```

Comment:
I left few comment for this on another pull request. Thank you 🙇‍♀️ for picking these up, it would be good to move one by one, will save your extra effort.